packages/extension-chakra-store-locator/src/setup-app.ts

```diff
@@ -35,7 +36,7 @@ class StoreLocatorExtension extends ApplicationExtension<Config> {
             )
         }
 
-        return withStoreLocator(withOptionalChakra(App), config)
+        return withStoreLocator(withOptionalCommerceSdkReactProvider(withOptionalChakra(App), config), config)
```

Comment:
There is a utility in the extensibility sdk that allows you to apply multiple hoc's. `import {applyHOCs} from '@salesforce/pwa-kit-extension-sdk/react/utils'` Can you give that a shot? It's possible that it might require some modification to allow each hoc to provide it's own configuration. But I think it's worth it since extension developers will end up using that utility .